1032-tabs-design-guidance.md

```diff
@@ -0,0 +1,39 @@
+---
+category: component design guidance
+state: draft
+created: 2024-10-18
+updated: 2024-10-21
+---
+
+# Tabs Design Guidance
+
+[Tabs](https://clarity.design/documentation/tabs) are components that organize the content of a page into multiple panes. Users can see the content of one tab at a time, yet quickly navigate to another tab.
+
+## Guidance
+
+- **Must** place tabs in a single, non-scrollable row, above their content.
+- **Must** avoid using tabs in cards and modals.
+- **Must** avoid using more than seven tabs and limit labels to one or two words to ensure that all tabs appear in the container. You may use [Tab Overflow](https://clarity.design/documentation/tabs/code#overflow-tabs) when the tab group is larger than its containing space or if you need to preserve space.
```

Comment:
```suggestion
- **Must** avoid using more than seven tabs and limit labels to one or two words to ensure that all tabs appear in the container. You **may** use [Tab Overflow](https://clarity.design/documentation/tabs/code#overflow-tabs) when the tab group is larger than its containing space or if you need to preserve space.
```

"May" should bolded. Also, please update the "must avoid" based on my other comment.